### ISSUE TITLE
Updating US Reporter Cites (one example, there may be others)

### ISSUE THREAD

**Issue by Colinstarger** [freelawproject/courtlistener#685]
In BECKLES (https://www.courtlistener.com/opinion/4374573/beckles-v-united-states/) the parser did not pick up reference to FCC v. Fox Broadcasting. The line in Beckles that contains the cite is: "The advisory Guidelines also do not implicate the vagueness doctrine’s concern with arbitrary enforcement. Laws that “regulate persons or entities,” we have explained, must be sufficiently clear “that those enforcing the law do not act in an arbitrary or discriminatory way.” FCC v. Fox Television Stations, Inc., 567 U.S. 239, 253 (2012).

The problem appears to be that CL's copy of FCC v. Fox does not have the US reporter cite associated with it. See https://www.courtlistener.com/opinion/802792/fcc-v-fox-television-stations-inc/. All of the cases that currently are recogized as citing FCC v. Fox use the S.Ct. reporter cite. 

This is potentially a larger issue because the US reporter cite generally takes a couple of years to get sorted out. Once it does get assigned, most courts prefer to cite to the US reporter. The lag means we may have cases -- like FCC v. Fox -- that only have SCT cites in the DB but now new courts are using the US reporter cite. 

Is there already a system in place to update SCOTUS cases so they have the US reporter cites?

**Comment by mlissner** [comment]
Yes, this is certainly an issue. If we had a good source for these citations, we could update the cases, but there's none that I know of. Maybe we need somebody with West access that can export a spreadsheet for us once in a while? Not sure how else it could work. 

It'd be nice, also, if this fix could also fix the other jurisdictions.

**Comment by mlissner** [comment]
One thing I'll mention is that I think this is theoretically possible to do just by mining the contents of opinions. For example, in your quote above we have all the information we need to add this citation to FCC v. Fox. This would only catch things after they were cited the first time, so boring cases would never get citations, but that's almost all that matters.

**Comment by mlissner** [comment]
Oh, and...if SCDB has this info and we import the latest SCDB, we'll get this info. That's how we got the four citations we already had.

**Comment by Colinstarger** [comment]
The SCDB import idea seemed perfect to me. But then I looked up FCC v. Fox on SCDB and they don't have the US citation! http://scdb.wustl.edu/analysisCaseListing.php?cite=S.%20Ct.&vol=132&page=2307 Grrrr

Suddenly I thought: could that have caused a problem? Do you both add citations and take away citations based on SCDB? Or only add?

**Comment by mlissner** [comment]
> Do you both add citations and take away citations based on SCDB? Or only add? 

Only add.

**Comment by mlissner** [comment]
See also: #608

**Comment by mlissner** [comment]
This looks like a winner for you @grossir! It should at least go on your backlog, but maybe you've even got a solution for it already?
